### PR TITLE
[#1253] bugfix(jdbc): fix jdbc catalog store empty audit info issue

### PR DESCRIPTION
**Pull request by Clearvive**
### What changes were proposed in this pull request?

Mysql&Postgres catalog store empty audit info fix.

### Why are the changes needed?
Fix: #1253 

### Does this PR introduce _any_ user-facing change?
Users will receive audit information

### How was this patch tested?
IT

### CODE REVIEW

**Comment by qqqttt123** [comment]
Will  this pull request  back port to branch 0.3?

**Comment by jerryshao** [comment]
@qqqttt123 please help to review.

**Review by qqqttt123 — APPROVED** [review]
LGTM, thanks @jerryshao @Clearvive

File: catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java [hunk 1]
```diff
@@ -276,7 +276,7 @@ public Table loadTable(NameIdentifier tableIdent) throws NoSuchTableException {
     }
     Map<String, String> properties =
         load.properties() == null ? Maps.newHashMap() : Maps.newHashMap(load.properties());
-    StringIdentifier.addToProperties(id, properties);
+    properties = StringIdentifier.addToProperties(id, properties);
```
Comment (qqqttt123) [inline catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java:279]: Could you rename `addToProperties` to a more proper name?

**Review by qqqttt123 — APPROVED** [review]

**Pull request merged**